finalPrep/modZeroHero.js

```diff
@@ -74,11 +142,46 @@ var excuse = 'I think I forgot to lock up my 1992 Toyota Coralla. Be right back.
 
 // - Create 2 instances of your SuperHero class
 
+class SuperHero {
+  constructor(name, superpower, age) {
+    this.name = name;
+    this.superpower = superpower;
+    this.age = age;
+  }
+    static archNemesis = "The Syntax Error"
+    static powerLevel = 100
+    static energyLevel = 50
+
+    sayName() {
+      console.log('It is me, ' + this.name + '!');
+    }
+    maximizeEnergy() {
+      SuperHero.energyLevel = 1000;
+    }
+    gainPower(number) {
+    SuperHero.powerLevel += number;
```

Comment:
Nice job making a dynamic function, but same as above comment about `this`, and don't forget to indent the content of your method!